recipes/influxdb-cpp/all/test_package/CMakeLists.txt

```diff
@@ -1,9 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
-project(PackageTest CXX)
+cmake_minimum_required(VERSION 3.15)
+project(test_package CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(influxdb-cpp REQUIRED CONFIG)
 
-add_executable(example example.cpp)
-target_link_libraries(example ${CONAN_LIBS})
-set_property(TARGET example PROPERTY CXX_STANDARD 11)
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package PRIVATE influxdb-cpp::influxdb-cpp)
+set_property(TARGET test_package PROPERTY CXX_STANDARD 11)
```

Comment:
Please check this comment: https://github.com/conan-io/conan-center-index/pull/18124#discussion_r1246361883